recipes/slac-rogue/recipe.yaml

```
@@ -0,0 +1,69 @@
+
+context:
+  version: "6.6.2"
+
+package:
+  name: slac-rogue
+  version: ${{ version }}
+
+source:
+  url: https://github.com/slaclab/rogue/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 77f0ed8c5f9e9e89f941d2b7fc3b5ec1e0c891ac002870cc9ccb8ead69a37efb
+build:
+  skip: true if not linux64
```

I think the correct way of writing this is just:
```suggestion
  skip: not linux64
```